admin/listPurchases/listView.html

```diff
@@ -98,18 +98,29 @@
 
 const amountOfShownSubpages = 5;
 const entriesPerSubpage = 20;
-var amountOfSubpages;
+let amountOfSubpages;
 
 function generateSubPageNavigation(){
-    nav.innerHTML = '';
-    orderSearchBar.value ? amountOfSubpages = Math.ceil(entries.filter(currentOption.filter).filter(searchBarFilter).length / entriesPerSubpage) : amountOfSubpages = Math.ceil(entries.filter(currentOption.filter).length / entriesPerSubpage);
-    if(entryPageIndex >= amountOfSubpages) entryPageIndex = amountOfSubpages - 1;
+    if(orderSearchBar.value){
+        amountOfSubpages = Math.ceil(entries.filter(currentOption.filter).filter(searchBarFilter).length / entriesPerSubpage);
+    }else{
+        amountOfSubpages = Math.ceil(entries.filter(currentOption.filter).length / entriesPerSubpage);
+    }
+
+    if(currentPageIndex >= amountOfSubpages){
+        currentPageIndex = amountOfSubpages - 1;
+    } 
+
     let startIndexToRender = 0;
-    if(amountOfSubpages > amountOfShownSubpages){
-        nav.innerHTML += `<a href="#" class="option subpage-selector subpage-selector-to-start"><<</a>`
-        nav.innerHTML += `<a href="#" class="option subpage-selector subpage-selector-backwards"><</a>`
+    let subpageNavigation = [];
+
+    const extraNavNeeded = amountOfSubpages > amountOfShownSubpages;
 
-        startIndexToRender = entryPageIndex;
+    if(extraNavNeeded){
+        subpageNavigation.push(`<a href="#" class="option subpage-selector subpage-selector-to-start">&lt&lt</a>`);
```

Comment:
I think `|<` indicates to start slightly better? Consider looking for a relevant UTF 8 arrow: https://www.w3schools.com/charsets/ref_utf_arrows.asp